curated_transformers/models/albert/layer_group.py

```diff
@@ -26,6 +27,12 @@ def __init__(
     ) -> None:
         super().__init__()
 
+        layer_norm = partial(
```

Comment:
Was this extracted into a `partial` just to avoid duplicate code in the `EncoderLayer` callsite?

Reply:
Yeah, shortens it a considerable bit. The lines are already pretty long, so the arguments get spread over multiple lines.